src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { ContainerRow } from '../../common/HelperComponents/ThemeHelper.styles';
+import { Icon } from './Icon';
+
+export default {
+  component: Icon,
+  parameters: {
+    componentSubtitle: 'An icon',
```

Comment:
Acho que aqui vale uma descrição tipo:
"A helper component to display icons from @naturacosmeticos-natds-icons"
que tal?